Fix not rendered icon and add example to override specific contenttype (#1689)
ezsystems/developer-documentation: docs/extending/extending_thumbnails.md

At this point you can go to the Back Office and check the results.

> !!! note "thumbnail mechanism "
> 
>     Above strategy will override all genertaed thumbnails. You might specify a specific contenttype. See example [here](https://github.com/ibexa/user/blob/main/src/lib/Strategy/DefaultThumbnailStrategy.php)
> 

Reviewer: ```suggestion
!!! note "Thumbnail mechanism "

    This strategy overrides all generated thumbnails. You can specify a specific Content Type. See the example [here](https://github.com/ibexa/user/blob/main/src/lib/Strategy/DefaultThumbnailStrategy.php)

```
